src/app/lists/combo/combo-binding/combo-binding.component.scss

```
@@ -0,0 +1,29 @@
+:host {
+    display: flex;
+    width: 100%;
+    flex-flow: row wrap;
+}
+
+button {
+    margin: 8px 0px;
+}
+
+.combo-section {
+    margin: 24px;
+    width: 320px;
+
+    label {
+        font-weight: 800;
+    }
+}
+
+.cities-container {
+    max-height: 200px;
+    overflow-y: auto;
+    margin: 8px 0px;
+
+    igx-card {
+        padding: 8px;
```

Padding was on the wrapper, for the card is margin (see unresolved comment from last review):
```suggestion
        margin: 8px 8px 8px 0;
```